### PR TITLE
[shopsys] admin design facelift

### PR DESCRIPTION
**Pull request by tomshopsys**
| Q             | A
| ------------- | ---
|Description, reason for the PR| administration wasn't well arranged and modern. This is just the first phase that doesn't include big changes in structure. There is preparation for the statistics on dashboard 
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| ... 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig [hunk 1]
```diff
@@ -1,38 +1,46 @@
 <ul class="js-toggle-menu navig navig--side">
     <li class="js-toggle-menu-item navig__item" >
-        <div
-                class="navig__item__link navig__item__link--bordered"
-                title="{{ 'Frontend'|trans }}"
-        >
-            <i class="svg svg-forward-page"></i>
+        <div class="box-dropdown ">
```
Comment (DavidOstrozlik) [inline packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig]:
```suggestion
        <div class="box-dropdown">
```

File: packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
-        </ul>
     </li>
 
-    <li class="js-toggle-menu-item navig__item" >
-        <div
-                class="navig__item__link navig__item__link--bordered"
-                title="{{ 'Account'|trans }}"
-        >
-            <i class="svg svg-person-man"></i>
+    <li class="js-toggle-menu-item navig__item navig__item" >
```
Comment (DavidOstrozlik) [inline packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig]:
```suggestion
    <li class="js-toggle-menu-item navig__item">
```

File: packages/framework/src/Resources/views/Admin/Menu/side_menu.html.twig [hunk 3]
```diff
@@ -111,4 +111,7 @@
     {%- else -%}
         {{ item.label }}
     {%- endif -%}
+
+        <i class="side-menu__item__arrow side-menu__item__arrow--down"></i>
```
Comment (DavidOstrozlik) [inline packages/framework/src/Resources/views/Admin/Menu/side_menu.html.twig]:
```suggestion
    <i class="side-menu__item__arrow side-menu__item__arrow--down"></i>
```

Comment (tomshopsys) [reply packages/framework/src/Resources/views/Admin/Menu/side_menu.html.twig]: thx for suggestions, fixed

**Pull request merged**